prometheus_client/exposition.py

```diff
@@ -377,10 +378,3 @@ def instance_ip_grouping_key():
     with closing(socket.socket(socket.AF_INET, socket.SOCK_DGRAM)) as s:
         s.connect(('localhost', 0))
         return {'instance': s.getsockname()[0]}
-
-
-try:
-    # Python >3.5 only
```

Comment:
Why are you removing this?

Reply:
Sorry, Deleted by mistake with auto flake.